soda_machine.rb

```diff
@@ -7,12 +7,23 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.count
   end
 
   def find_soda(soda_brand)
+    @sodas.find{|soda| return soda if soda.brand == soda_brand}
   end
 
```

Comment:
This can be condensed  to:

```ruby
  @sodas.find{|soda| soda.brand == soda_brand}
```
The method ```find``` automatically returns the first match.